hypervisor/arch/x86/guest/ucode.c

```diff
@@ -7,6 +7,8 @@
 #include <hypervisor.h>
 #include <ucode.h>
 
+#define MICRO_CODE_SIZE_MAX    0x40000U
+static uint8_t micro_code[MICRO_CODE_SIZE_MAX];
```

Comment:
I remember ucode update is per core. So, it is possible 2 VCPUs may update ucode in parallel. In this case, we need a lock, right?  Try to see if we can reuse an existing lock..

Reply:
can not find an existing lock,  will add anew